powershell/models/TrieNode.ts

```diff
@@ -0,0 +1,40 @@
+export class TrieNode {
```

Comment:
Please add license header for this file and move it to utils folder.

Reply:
fixed